src/main/kotlin/org/damascus/Main.kt

```diff
@@ -7,7 +7,7 @@ import org.damascus.ui.PlanMateConsoleUi
 import org.koin.core.context.startKoin
 import org.koin.java.KoinJavaComponent.getKoin
 
-fun main() {
+suspend fun main() {
```

Comment:
Why did you choose to make main suspend-able? instead of using a Co-routine scope where you would need to call a suspend-able function?